{"variants":[{"paths":["\/documentation\/axpcore\/axperror\/invalidstate"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore","doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore\/AXPError"]]},"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore\/AXPError\/invalidState"},"metadata":{"roleHeading":"Case","role":"symbol","externalID":"s:7AXPCore8AXPErrorO12invalidStateyA2CmF","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"invalidState","kind":"identifier"}],"symbolKind":"case","title":"AXPError.invalidState","modules":[{"name":"AXPCore"}]},"schemaVersion":{"major":0,"patch":0,"minor":3},"kind":"symbol","abstract":[{"type":"text","text":"If the operation is not valid for the current state"}],"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"invalidState"}]}],"kind":"declarations"}],"references":{"doc://com.avaya.axp.client.sdk.AXPCore/documentation/AXPCore/AXPError/invalidState":{"abstract":[{"text":"If the operation is not valid for the current state","type":"text"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidState","kind":"identifier"}],"title":"AXPError.invalidState","type":"topic","url":"\/documentation\/axpcore\/axperror\/invalidstate","role":"symbol","kind":"symbol","identifier":"doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore\/AXPError\/invalidState"},"doc://com.avaya.axp.client.sdk.AXPCore/documentation/AXPCore":{"abstract":[{"text":"The base module that establishes the session with Avaya Experience Platformâ„¢ (AXP) for the end user.","type":"text"}],"type":"topic","identifier":"doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore","kind":"symbol","title":"AXPCore","url":"\/documentation\/axpcore","role":"collection"},"doc://com.avaya.axp.client.sdk.AXPCore/documentation/AXPCore/AXPError":{"type":"topic","navigatorTitle":[{"kind":"identifier","text":"AXPError"}],"abstract":[{"type":"text","text":"Error for the case where an operation failed due to an error contacting AXP"},{"type":"text","text":" "},{"text":"across the network.","type":"text"}],"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AXPError","kind":"identifier"}],"kind":"symbol","title":"AXPError","identifier":"doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore\/AXPError","url":"\/documentation\/axpcore\/axperror"}}}